README.md

```diff
@@ -356,6 +356,7 @@ Or you can just run: `./ftw run --cloud`
 ## How log parsing works
 
 The WAF's log file with the alert messages is parsed and compared to the expected output defined in the unit test under `log_contains` or `no_log_contains`.
+Note that the expected output may contain multiple checks (E.g. `log_contains` and `status`). If any of the checks fail, the test will fail.
```

Comment:
Address nit https://github.com/coreruleset/go-ftw/pull/139#pullrequestreview-1336083730. Not sure about it is the best spot 🧐

Reply:
It is good.